pkg/agent/agent.go

```diff
@@ -188,6 +203,7 @@ func (a *Agent) newSVID(key *ecdsa.PrivateKey, bundle []*x509.Certificate) (*x50
 		Csr:          csr,
 	}
 	resp, err := c.FetchBaseSVID(context.TODO(), req)
+
```

Comment:
was this extra newline intentional? Usually error checks come immediately after the value is received

Reply:
You are right, this extra newline doesn't make sense...I will remove it.